server.js

```diff
@@ -5,6 +5,11 @@ if (typeof userFunctionUri === "undefined" || userFunctionUri.trim() === "") {
     throw "FUNCTION_URI envvar not set or empty. Aborting.";
 }
 
+const debug = process.env.NODE_DEBUG;
+if (typeof debug === "undefined" || debug !== "riff") {
+    console.debug = () => {};
```

Comment:
this should be the very first thing to happen, even before requiring other modules in this package in case they need to log.